tests/unit/samples/db_files/init.sql

```diff
@@ -27,3 +27,26 @@ CREATE TABLE IF NOT EXISTS "fake_table" (
   "rank" INTEGER NOT NULL,
   PRIMARY KEY("id")
 );
+
+CREATE TABLE IF NOT EXISTS "mail_group" (
```

Comment:
wait... are we copying the database from database to samples? We should use the file from the database folder; otherwise, we might create differences between these two files.